### DISCUSSION TITLE
Add new function: Search all tables

### DISCUSSION THREAD

**Discussion by avcommunication** [Sequel-Ace/Sequel-Ace#1514] [Ideas]
Hello everybody,
I invite you to think of a new feature that could be very, very convenient and interesting.

It is possible to foresee a new icon in the header next to that of TABLE INFO or QUERY: SEARCH ALL

By inserting the search key, the text will be searched in all fields of all tables. The same function can also be inserted by selecting a specific table.

This function can save you a lot of time if you are looking for content within a table with many columns.

Is it something that can be installed?

**Comment by Jason-Morcos** [comment]
This has been requested in many ways many times. I'm not technically sure how to implement it in a way that's performant. While it might make sense in a tiny database, on a production level database with hundreds of millions of records this seems like a recipe for disaster. PRs are always welcome if someone has an idea for how to implement this in a performant fashion at scale.

See the currently-open-issue where this has been requested. https://github.com/Sequel-Ace/Sequel-Ace/issues/152
